plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -226,6 +226,12 @@ bool GSDevice11::Create(const std::shared_ptr<GSWnd> &wnd)
 		return false;
 	}
 
+	// Set maximum texture size limit based on supported feature level.
+	if (level < D3D_FEATURE_LEVEL_11_0)
```

Comment:
I find it more readable when the conditions are inverted, D3D11+ cases at first and the lower ones as fallback. Do you agree?